### ISSUE TITLE
Warning: Calling fails_with :llvm is deprecated!

### ISSUE THREAD

**Issue by Snuff** [staticfloat/homebrew-julia#242]
Hey, as I see you use a deprecated method.

```
Warning: Calling fails_with :llvm is deprecated!
There is no replacement.
/usr/local/Homebrew/Library/Taps/staticfloat/homebrew-julia/llvm37-julia.rb:166:in `<class:Llvm37Julia>'
Please report this to the staticfloat/julia tap!
```

**Comment by Snuff** [comment]
ping @staticfloat

**Comment by staticfloat** [comment]
Addressed in 7caf4d5.  Thanks!
